src/Tape.cpp

```diff
@@ -712,13 +711,13 @@ void Tape<T>::clearDerivativesAfter(position_type pos)
 }
 
 template <class T>
-T Tape<T>::getAndResetOutputAdjoint(slot_type slot)
-{
-    if (slot >= slot_type(derivatives_.size()))
+T Tape<T>::getAndResetOutputAdjoint(slot_type slot) {
+    const auto derivatives_size = derivatives_.size();
+    if (slot >= slot_type(derivatives_size))
```

Comment:
Why did you make that change? As far as we can see, it doesn't change anything...

Also, could you please use clang-format for code formatting, to ensure consistent style? For example we put the opening curly brace on a line of its own.